index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome Page</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body >
+    <header class="flexBox" >
+
+        <h1>To-Do</h1>
+        <img src="/assets/true.png" alt="logo">
+        <p>A simple and intellgent to-do list makes it easy to plan your day</p>
+
+    </header>
+    <footer class="flexBox2">
+        <button>Start</button>
```

Comment:
Do not forget linking to Home Page
my suggeestion is:
`<button onclick="window.location.href='/html/home.html';">Start</button>`
If there better way do it